scripts/install_debugadapter.ps1

```diff
@@ -1,3 +1,3 @@
 dotnet pack ../src/Draco.DebugAdapter --output .
-try { dotnet tool uninstall --global Draco.DebugAdapter } catch {}
+try { dotnet tool uninstall --global Draco.DebugAdapter --verbosity quiet } catch {}
```

Comment:
Any reason for this? Won't this silence the error in case the tool is running and it can't uninstall?
I do that pretty common when I forget that VSC is running already

Reply:
Well the other scripts use it in the same exact fashion.